From 19bcf1c1f550e964970347c27ee7e1694c502783 Mon Sep 17 00:00:00 2001
From: sorlandini <sandrine.orlandini@hach.com>
Date: Fri, 7 Jul 2017 09:40:02 +0200
Subject: [PATCH] Put struct definitions before using them

---
 plugins/telit-serial.h | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/plugins/telit-serial.h b/plugins/telit-serial.h
index f74bcf1..6e09b06 100644
--- a/plugins/telit-serial.h
+++ b/plugins/telit-serial.h
@@ -23,15 +23,6 @@
 #define CMUX_CLOSE_DELAY 3*1000*1000 /* uS */
 #define CMUX_DELAY 3*1000*1000 /* uS */
 
-static const char *none_prefix[] = { NULL };
-pthread_t tid;
-static GSList *modem_list = NULL;
-
-void* DeviceDetectionThread(void *arg);
-static void modem_create();
-static void modem_remove();
-enum cmux_state_t cmux_state;
-
 struct telit_serial_data {
 		GAtChat *chat;		/* AT chat */
 		GAtChat *modem;		/* Data port */
@@ -58,5 +49,14 @@ const char* CMUX_STATE_STRING[] =
 	"CMUX_STATE_UNKNOWN", // 4
 };
 
+static const char *none_prefix[] = { NULL };
+pthread_t tid;
+static GSList *modem_list = NULL;
+
+void* DeviceDetectionThread(void *arg);
+static void modem_create();
+static void modem_remove();
+enum cmux_state_t cmux_state;
+
 inline const char* cmux_string(enum cmux_state_t cmux_s);
 int file_exist (char *filename);
-- 
2.7.4

