From 57a4f6bff908596250385461473b825c096c1a61 Mon Sep 17 00:00:00 2001
From: sorlandini <sandrine.orlandini@hach.com>
Date: Tue, 6 Mar 2018 14:00:47 +0100
Subject: [PATCH] Change Ofono version to include fix USB descriptor

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 5620fec..9da05ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1,5 +1,5 @@
 AC_PREREQ(2.60)
-AC_INIT(ofono, 1.18_R5.00.02.B3)
+AC_INIT(ofono, 1.18_R5.00.02.B3_1)
 
 AM_INIT_AUTOMAKE([foreign subdir-objects color-tests])
 AC_CONFIG_HEADERS(config.h)
-- 
2.7.4

